connectivity/drivers/lora/TARGET_STM32WL/STM32WL_radio_driver.cpp

```diff
@@ -16,8 +16,49 @@
 #include "STM32WL_radio_driver.h"
 #include "drivers/DigitalOut.h"
 
-
 /* Sets up radio switch position according to the radio mode */
+#if defined (TARGET_LORA_E5) || defined (TARGET_RAK3172)
+/* This configuration is for RAK3172 or LoRa-E5 modules */
+/* Theese one use only HP mode the LP mode is not connected nor RF Switch ctrl3 */
```

Comment:
typo issues

Reply:
could you precise what do you want me to fix?

Reply:
done